samples/openthread/cli/common.cmake

```diff
@@ -0,0 +1,97 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+# Header guard
+if(__OT_CLI_COMMON_CMAKE__)
+  return()
+endif()
+set(__OT_CLI_COMMON_CMAKE__ TRUE)
+
+include(${NRFXLIB_DIR}/common.cmake)
+
+# Copy new OpenThread libraries, headers, text file with the configuration
+# to the nrfxlib openthread folder.
+function(ot_libs_substitute)
+  execute_process(
+    COMMAND           git rev-parse HEAD
```

Comment:
Use `${GIT_EXECUTABLE}` instead.

Secondly, `git` is not mandatory for building, so please ensure code is working, even if `git` is not available.

Reply:
Done.